docs/manual/cluster-admin/installation-guide.md

```diff
@@ -167,10 +167,10 @@ git checkout v1.5.0
 ```
 
 Please edit `layout.yaml` and a `config.yaml` file under `<pai-code-dir>/contrib/kubespray/config` folder.
-These two files spedify the cluster layout and the customized configuration, respectively.
+These two files specify the cluster layout and the customized configuration, respectively.
 The following is the format and example of these 2 files.
 
-**Tips for China Users**: If you are a China user, before you edit these files, please refer to [here](./configuration-for-china.md) first.
+**Tips for China Users**: If you are in China, please refer to [here](./configuration-for-china.md) first before you edit these files.
```

Comment:
"Chinese Users"; "in Mainland China"?

Reply:
Hi Yifan, I updated the docs according to your suggestion. Can you review it again?